supertokens_python/recipe/passwordless/recipe.py

```diff
@@ -56,12 +56,12 @@ class PasswordlessRecipe(RecipeModule):
 
     def __init__(self, recipe_id: str, app_info: AppInfo, contact_config: ContactConfig,
                  flow_type: Literal['USER_INPUT_CODE', 'MAGIC_LINK', 'USER_INPUT_CODE_AND_MAGIC_LINK'],
+                 ingredients: PasswordlessIngredients,
                  override: Union[OverrideConfig, None] = None,
                  get_link_domain_and_path: Union[Callable[[
                      PhoneOrEmailInput, Dict[str, Any]], Awaitable[str]], None] = None,
                  get_custom_user_input_code: Union[Callable[[Dict[str, Any]], Awaitable[str]], None] = None,
                  email_delivery: Union[EmailDeliveryConfig[TypePasswordlessEmailDeliveryInput], None] = None,
-                 ingredients: Union[PasswordlessIngredients, None] = None,
```

Comment:
Why this change? And why did you make it non optional?

Reply:
We are keeping `ingredients` non-optional in all the recipes.